tests/Aspire.Hosting.Tests/Azure/AzureEventHubsExtensionsTests.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Utils;
+using Aspire.Hosting.Azure.EventHubs;
+using Xunit;
+
+namespace Aspire.Hosting.Tests.Azure;
+
+public class AzureEventHubsExtensionsTests
+{
+    [Fact]
+    public void AzureEventHubsUseEmulatorCallbackWithWithDataBindMountResultsInBindMountAnnotationWithDefaultPath()
+    {
+        using var builder = TestDistributedApplicationBuilder.Create();
+        var eventHubs = builder.AddAzureEventHubs("eh").RunAsEmulator(configureContainer: builder =>
+        {
+            builder.WithDataBindMount();
+        });
+
+        // Ignoring the annotation created for the custom Config.json file
+        var volumeAnnotation = eventHubs.Resource.Annotations.OfType<ContainerMountAnnotation>().Where(a => !a.Target.Contains("Config.json")).Single();
```

Comment:
(nit) repeated elsewhere.

```suggestion
        var volumeAnnotation = eventHubs.Resource.Annotations.OfType<ContainerMountAnnotation>().Single(a => !a.Target.Contains("Config.json"));
```